compiler/compile/OMRCompilation.hpp

```diff
@@ -969,15 +965,15 @@ class OMR_EXTENSIBLE Compilation
    void reportFailure(const char *reason);
 
 protected:
+   // Signature of the method we are compiling -- keep this at offset 0
```

Comment:
It's not clear to me whether this is a hard requirement driven by some downstream project built on OMR or merely a suggestion.  There are certainly examples of other places in OMR where a downstream project has built tooling (and hence an undocumented dependence) on the location of a particular field in a data structure.  This is wrong in my opinion.

I am more in favour of removing the comment entirely, especially if its only a "soft" requirement whose presence will likely discourage future refactoring.  If it truly is a "hard" requirement then it should be documented as such and properly justified in comments in OMR.